### PR TITLE
Add a set of Arktos-CI test mark

### PR DESCRIPTION
**Pull request by Hong-Chang**
Mark a bunch of test as runable by arktos CI

**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
To run the selected test by the mark, use: --test_args="--ginkgo.focus=\[Arktos-CI\]"

### CODE REVIEW

File: test/e2e/common/expansion.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ var _ = framework.KubeDescribe("Variable Expansion", func() {
 		Testname: Environment variables, command expansion
 		Description: Create a Pod with environment variables and container command using them. Container command using the  defined environment variables MUST expand to proper values.
 	*/
-	framework.ConformanceIt("should allow substituting values in a container's command [NodeConformance]", func() {
+	framework.ConformanceIt("should allow substituting values in a container's command [NodeConformance] [Arktos-CI]", func() {
```
Comment (Sindica) [inline test/e2e/common/expansion.go:91]: This does not look like essential test case. Please verify with Xiaoning.

File: test/e2e/common/init_container.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ var _ = framework.KubeDescribe("InitContainer [NodeConformance]", func() {
 		and at least one container is still running or is in the process of being restarted
 		when Pod has restart policy as RestartAlways.
 	*/
-	framework.ConformanceIt("should invoke init containers on a RestartAlways pod", func() {
+	framework.ConformanceIt("should invoke init containers on a RestartAlways pod [Arktos-CI]", func() {
```
Comment (Sindica) [inline test/e2e/common/init_container.go:124]: Same here

File: test/e2e/common/runtime.go [hunk 3]
```diff
@@ -44,7 +44,7 @@ var _ = framework.KubeDescribe("Container Runtime", func() {
 				Testname: Container Runtime, Restart Policy, Pod Phases
 				Description: If the restart policy is set to ‘Always’, Pod MUST be restarted when terminated, If restart policy is ‘OnFailure’, Pod MUST be started only if it is terminated with non-zero exit code. If the restart policy is ‘Never’, Pod MUST never be restarted. All these three test cases MUST verify the restart counts accordingly.
 			*/
-			framework.ConformanceIt("should run with the expected status [NodeConformance]", func() {
+			framework.ConformanceIt("should run with the expected status [NodeConformance] [Arktos-CI]", func() {
```
Comment (Sindica) [inline test/e2e/common/runtime.go:47]: May not be essential test case

File: test/e2e/kubectl/kubectl.go [hunk 4]
```diff
@@ -1405,7 +1405,7 @@ metadata:
 			Testname: Kubectl, version
 			Description: The command ‘kubectl version’ MUST return the major, minor versions,  GitCommit, etc of the Client and the Server that the kubectl is configured to connect to.
 		*/
-		framework.ConformanceIt("should check is all data is printed ", func() {
+		framework.ConformanceIt("should check is all data is printed [Arktos-CI]", func() {
```
Comment (Sindica) [inline test/e2e/kubectl/kubectl.go:1408]: same here

File: test/e2e/common/expansion.go (hunk 1 above)
Comment (Hong-Chang) [reply test/e2e/common/expansion.go:91]: Removed

File: test/e2e/common/init_container.go (hunk 2 above)
Comment (Hong-Chang) [reply test/e2e/common/init_container.go:124]: Removed

File: test/e2e/common/runtime.go (hunk 3 above)
Comment (Hong-Chang) [reply test/e2e/common/runtime.go:47]: In runtime area it's pretty representative. Better than other runtime cases.

File: test/e2e/kubectl/kubectl.go (hunk 4 above)
Comment (Hong-Chang) [reply test/e2e/kubectl/kubectl.go:1408]: Changed to another two more essential cases.

File: test/e2e/kubectl/kubectl.go [hunk 5]
```diff
@@ -284,7 +284,7 @@ var _ = SIGDescribe("Kubectl client", func() {
 			Testname: Kubectl, replication controller
 			Description: Create a Pod and a container with a given image. Configure replication controller to run 2 replicas. The number of running instances of the Pod MUST equal the number of replicas set on the replication controller which is 2.
 		*/
-		framework.ConformanceIt("should create and stop a replication controller ", func() {
+		framework.ConformanceIt("should create and stop a replication controller [Arktos-CI]", func() {
```
Comment (Sindica) [inline test/e2e/kubectl/kubectl.go:287]: Do we have a test for replicaset controller or deployment controller? Replication Controller is deprecated

Comment (Hong-Chang) [reply test/e2e/kubectl/kubectl.go:287]: Currently I'm adding them. I'm using the preset machines to verify.

File: test/e2e/common/runtime.go (hunk 3 above)
Comment (Sindica) [reply test/e2e/common/runtime.go:48]: This is a test for kubectl. Try not use deprecated resource. It feels like "Kubectl run deployment" would be a better core test.

File: test/e2e/kubectl/kubectl.go (hunk 5 above)
Comment (Sindica) [reply test/e2e/kubectl/kubectl.go:287]: This is a test for kubectl. Try not use deprecated resource. It feels like "Kubectl run deployment" would be a better core test.

File: test/e2e/common/runtime.go (hunk 3 above)
Comment (Sindica) [reply test/e2e/common/runtime.go:48]: "when running a container with a new image" might be better candidate for core test.

Comment (Hong-Chang) [reply test/e2e/common/runtime.go:48]: "when running a container with a new image" is context, not a case. Under the context, I added one test case. Will show in next iteration.

File: test/e2e/kubectl/kubectl.go (hunk 5 above)
Comment (Hong-Chang) [reply test/e2e/kubectl/kubectl.go:287]: I switched to another case. Will show in next iteration.

**Comment by Sindica** [comment]
Question: how long does it take to run all the chosen tests?

**Pull request merged**